assets/src/edit-story/elements/media/display.js

```diff
@@ -36,7 +36,7 @@ import { useUnits } from '../../units';
 import { getMediaWithScaleCss } from './util';
 import getMediaSizePositionProps from './getMediaSizePositionProps';
 
-const Element = styled.div`
+const Element = styled.div.attrs({ className: 'dashboard-media-element' })`
```

Comment:
Why `dashboard`? We're in the editor :-)

Perhaps `story-media-display-element`?